### PR TITLE
refactor(sqlite): fix some warnings

### PR DESCRIPTION
**Pull request by open-schnick**

### CODE REVIEW

File: src/db/sqlite/src/lib.rs [hunk 1]
```diff
@@ -70,17 +70,13 @@ impl Driver for Sqlite {
         let mut stmt = connection.prepare(&sql_str).unwrap();
 
         if ty.is_none() {
-            let exec = match sql {
-                sql::Statement::Update(u) if u.pre_condition.is_some() => false,
-                _ => true,
-            };
+            let exec = !matches!(sql, sql::Statement::Update(u) if u.pre_condition.is_some());
```
Comment (open-schnick) [inline src/db/sqlite/src/lib.rs:73]: maybe one could rename the variable to make it express the actual meaning behind not executing the query in some cases

Comment (carllerche) [reply src/db/sqlite/src/lib.rs:73]: sure!

Comment (open-schnick) [reply src/db/sqlite/src/lib.rs:73]: actually i am not really sure what the code means ^^
we dont exec when the sql is not an `Statement::Update` or an `Statement::Update` without a `pre_condition`
renaming exec to something more meaningful would make the code more readable i think

Comment (carllerche) [reply src/db/sqlite/src/lib.rs:73]: When `exec` is true, call `stmt.execute`. When `exec` is false, call `stmt.query`). (i.e true is call the execute method).

**Pull request merged**